.eslintrc

```diff
@@ -139,7 +142,7 @@
         "valid-jsdoc": [
             "error",
             {
-                "matchDescription": ".+"
```

Comment:
@noxi333 can you revert this change, this will result in erros with multiline descriptions (of which one is in the frontend already)